src/ntrip_client/ntrip_client.py

```diff
@@ -23,6 +24,13 @@
 
 class NTRIPClient:
 
+  _basic_credentials = None
+  _connected = False
+  _valid_nmea = False
+  _lastNEMEA = None
```

Comment:
These variables should be snake case, and `NEMEA` should be `NMEA` unless `NEMEA` is an abbreviation I am unaware of, for example: `_last_nmea`

Reply:
yep :-)